src/components/List/List.tsx

```diff
@@ -13,6 +13,7 @@ import SortHeader, { SortHeaderProps } from './components/SortHeader';
 import ListGroup from './ListGroup';
 import ListGroupItem from './ListGroupItem';
 import ListItem, { ListItemProps } from './ListItem';
+import Label from '../Label/Label';
```

Comment:
Let's move this per the warning 👍

Reply:
That's weird, the warning only shows when I view all changes so I totally missed it